rc/filetype/org.kak

```diff
@@ -0,0 +1,262 @@
+# Options
+add-highlighter shared/org/inline/text/option     regex "(?i)#\+[a-z]\w*\b[^\n]*"          0:module
+add-highlighter shared/org/inline/text/title      regex "(?i)#\+title:([^\n]+)"            0:module 1:title
+add-highlighter shared/org/inline/text/requisites regex "(?i)#\+(?:author|email):([^\n]+)" 0:module 1:keyword
+
+# Drawer
+add-highlighter shared/org/inline/text/drawer   regex "^\h*([:][^\s][^\n]*?[^\s]*?[:])\W" 1:keyword
+
+# Timestamps
+declare-option -docstring "Org date
+          format: YYYY-MM-DD DAYNAME" \
+regex org_date '\d\d\d\d-\d\d-\d\d\h+[^\s-+>\]\d]+'
+
+declare-option -docstring "Org time
+          format H:MM" \
+regex org_time '([0-2])?[0-9]:[0-5][0-9]'
+
+declare-option -docstring "Org repeater or delay
+          format: (.+|++) or (-|--) digit (hour|day|week|month|year)
+" \
+regex org_repeater_or_delay '([.+][+]|[-]{1,2})\h+\d\h+[hdwmy]'
+
+declare-option -docstring "Org timestamp
+          format: DATE TIME REPEATER-OR-DELAY" \
+regex org_timestamp "%opt{org_date}(\h+%opt{org_time}(-%opt{org_time})?(\h+%opt{org_repeater_or_delay})?)?"
+
+add-highlighter shared/org/inline/text/timestamp dynregex "[\[<]%opt{org_timestamp}([\]>]--[\[<]%opt{org_timestamp})?[\]>]" 0:variable
+
+# Markup
+## FORMAT: PRE MARKER `CONTENTS` MARKER POST
+## PRE: whitespace, `(`, `{`, `'`, `"`, or beginning of the line
+## MARKER: `/`, `+`, `=`, `~`, `_`, or `$`
+## CONTENTS FORMAT: BORDER BODY BORDER
+### BORDER: any non-whitespace, and not `,`, `'`, or `"`
+### BODY: any character, can't be longer than tho lines
+### POST: a whitespace character, `-`, `.`, `,`, `:`, `!`, `?`, `'`, `)`, `}`
+add-highlighter shared/org/inline/text/italic         regex "(^|[\h({'""])([/][^\h,'""][^\n]*?(\n[^\n]*?[^,'""\s])?[/])[\s.,:!?')}]" 2:org_italic
+add-highlighter shared/org/inline/text/strikethrough  regex "(^|[\h({'""])([+][^\h,'""][^\n]*?(\n[^\n]*?[^,'""\s])?[+])[\s.,:!?')}]" 2:org_strikethrough
+add-highlighter shared/org/inline/text/verbatim       regex "(^|[\h({'""])([=][^\h,'""][^\n]*?(\n[^\n]*?[^,'""\s])?[=])[\s.,:!?')}]" 2:meta
+add-highlighter shared/org/inline/text/code           regex "(^|[\h({'""])([~][^\h,'""][^\n]*?(\n[^\n]*?[^,'""\s])?[~])[\s.,:!?')}]" 2:mono
+add-highlighter shared/org/inline/text/underlined     regex "(^|[\h({'""])([_][^\h,'""][^\n]*?(\n[^\n]*?[^,'""\s])?[_])[\s.,:!?')}]" 2:org_underline
+## will be deprecated in future releases of Org but currently is supported
+add-highlighter shared/org/inline/text/inline-math    regex "(^|[\h({'""])([$][^\h,'""][^\n]*?(\n[^\n]*?[^,'""\s])?[$])[\s.,:!?')}]" 2:mono
+## bold is kinda tricky because we need to HL everything but headings, so it's split up on several regexps
+add-highlighter shared/org/inline/text/bold regex "(?:^|[\h({'""])([*][^\h,'""][^\n]*?(\n[^\n]*?[^,'""\s])?[*])[\s.,:!?')}]|([*]{3,})\n|\h([*]{3})[\s.,:!?')}]" 1:org_bold
+
+# LaTeX
+add-highlighter shared/org/LaTeX region -match-capture '\\begin\{([A-Za-z0-9*]+)\}' '\\end\{([A-Za-z0-9*]+)\}' fill string
```

Comment:
Maybe you could just reference the LaTeX highlighter directly here (`ref latex`)?

Reply:
that's nice idea, I'll try